### PR TITLE
Feature/tao 5239 brs integration

### PR DESCRIPTION
**Pull request by krampstudio**
Add a multiple delete action, to work with the resource selector.
Refresh the actionManager to select the correct delivery after assembling.

### CODE REVIEW

File: views/js/controller/DeliveryMgmt/wizard.js [hunk 1]
```diff
@@ -61,16 +60,14 @@ define([
     };
 
     /**
-     * wrapped the old jstree API used to refresh the tree and optionally select a resource
+     * wrapped the old jstree API used to refresh the tree and optionally select a resour
```
Comment (jsconan) [inline views/js/controller/DeliveryMgmt/wizard.js]: small typo here

File: views/js/controller/DeliveryMgmt/wizard.js [hunk 2]
```diff
@@ -61,16 +60,14 @@ define([
...
      * @param {String} [uriResource] - the uri resource node to be selected
      */
     var refreshTree = function refreshTree(uriResource){
-        var currentSection = section.current().selected;
-        var $trees = (currentSection && currentSection.panel && currentSection.panel.length) ?
-            $('.tree', currentSection.panel) : $('.tree:visible');
-        $trees.trigger('refresh.taotree', [{
-            selectNode : uriHelper.encode(uriResource)
-        }]);
+        console.log(uriResource);
```
Comment (jsconan) [inline views/js/controller/DeliveryMgmt/wizard.js]: it is intentional?

**Comment by krampstudio** [comment]
Updated.

**Comment by krampstudio** [comment]
I don't know why this branch has so much merge commits

File: controller/structures.xml [hunk 3]
```diff
@@ -15,36 +15,32 @@
...
-                    <action id="delivery-search" name="Search" url="/taoDeliveryRdf/DeliveryMgmt/search" context="*" group="search" binding="launchFinder">
-                        <icon id="icon-find"/>
-                    </action>  
                     <action id="delivery-class-new" name="New class" url="/taoDeliveryRdf/DeliveryMgmt/addSubClass" context="resource" group="tree" binding="subClass">
                         <icon id="icon-folder-open"/>
                     </action>
                     <action id="delivery-delete" name="Delete" url="/taoDeliveryRdf/DeliveryMgmt/delete" context="resource" group="tree" binding="removeNode">
                         <icon id="icon-bin"/>
                     </action>
+                    <action id="delivery-delete-all" name="Delete" url="/taoDeliveryRdf/DeliveryMgmt/deleteAll" context="resource" multiple="true" group="tree" binding="removeNodes">
```
Comment (jsconan) [inline controller/structures.xml]: The display name of 'delete all' is the same as for the 'delete one'.

**Pull request merged**